jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -325,8 +338,9 @@ public int getPageSize() {
 	protected Map<String, String> decodeCredentials(String jiraBasicAuthCredentialsInBase64) {
 		Map<String, String> credMap = new LinkedHashMap<String, String>();
 		if (jiraBasicAuthCredentialsInBase64 != null) {
+				//the tokenize includes a \n to ensure we trim those off the end (mac base64 adds these!)
 			StringTokenizer tokenizer = new StringTokenizer(new String(
-					Base64.decodeBase64(jiraBasicAuthCredentialsInBase64)), ":");
+					Base64.decodeBase64(jiraBasicAuthCredentialsInBase64)), ":\n");
```

Comment:
Will this not add an additional string token by doing this (e.g., a username, password, and a potential other token with a return carriage in it)?  The next section of code (lines 330-336) will loop through the tokens; please ensure this scenario will not cause a downstream ArrayIndexOutOfBoundsException; at minimum, please prove this with a basic JUnit test (note: there is already a small test that evaluates this class which you can append to).